frontend/src/components/contacts.tsx

```diff
@@ -1,57 +1,59 @@
-import React, { useState } from "react";
-import axios from "axios";
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
let's try to avoid file-level rule disables. 99% of the time the rules should just be either turned off or appropriately calibrated in the eslint. the remaining exceptions should be excluded at the line level, not at the file level

Reply:
rm'ed